share/templates/08-Orchestral/02-Symphony_Orchestra.mscx

```diff
@@ -1140,7 +1141,7 @@
           <program value="46"/>
           <synti>Fluid</synti>
           </Channel>
-        <Channel name ="staccato">
+        <Channel name="staccato">
```

Comment:
outch that was a bad one

Reply:
Bad? Do you mean this change is bad?

Reply:
yes, good to see this fixed

Reply:
I don't see a difference. Just a space, which is allowed in XML...

Reply:
In that case... still better to have it fixed